### PR TITLE
CLOUD-3180 - update EAP SSO Adapters to RH-SSO 7.3

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3180
Signed-off-by: Ken Wills <kwills@redhat.com>

### CODE REVIEW

**Comment by luck3y** [comment]
@ricardozanini FYI. Once this goes in to master we'll need to cherry pick the commits into the 7.2.x branch to get it into stable. (Same for 6.4.x).

**Comment by bstansberry** [comment]
@luck3y the change looks fine but CLOUD-3180 is for 7.2, not master.

**Comment by luck3y** [comment]
@bstansberry Update the Jira to drop the target release - it doesn't have a 7.2.x label, so its for master. We'll clone to 7.2.x and 6.4.x from that one.

**Comment by ricardozanini** [comment]
@luck3y I think we might have to update again?
https://access.redhat.com/errata/RHSA-2019:1140
:/

**Comment by luck3y** [comment]
@ricardozanini Yeah, will update this with the updated ones, thanks!

File: os-eap64-sso/module.yaml [hunk 1]
```diff
@@ -7,9 +7,10 @@ execute:
...
-  description: "The artifact is available on the Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=57851&product=core.service.rhsso&version=7.2&downloadType=securityPatches"
-  sha256: 739f2e92615bfd874c4b2c7fd8da2d5d4d8560bd853d459f9b9fcb131becb8f9
-- path: rh-sso-7.2.2-saml-eap6-adapter.zip
-  description: "The artifact is available on the Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=57811&product=core.service.rhsso&version=7.2&downloadType=securityPatches"
-  sha256: 27ace37d175ef3023cd5aa0dcdff8e54c86fc803fc8ef38d1ab7240cb006628a
+- name: rh-sso-7.3-eap6-adapter
+  target: rh-sso-7.3-eap6-adapter.zip
+  md5: dddbebeb28da106e7d4ff3e5a800fbe5
+- name: rh-sso-7.3-saml-eap6-adapter.zip
+- target: rh-sso-7.3-saml-eap6-adapter.zip
```
Comment (ricardozanini) [inline os-eap64-sso/module.yaml]: @luck3y this dash is causing this error:
`Resource type is not supported: {'name': 'rh-sso-7.3-saml-eap6-adapter.zip'}`

Also, the name of the artifact can be just `rh-sso-7.3-saml-eap6-adapter`?

Comment (ricardozanini) [inline os-eap64-sso/module.yaml]: @luck3y this dash is causing this error:
`Resource type is not supported: {'name': 'rh-sso-7.3-saml-eap6-adapter.zip'}`

Also, the name of the artifact can be just `rh-sso-7.3-saml-eap6-adapter`?

Comment (luck3y) [reply os-eap64-sso/module.yaml]: @ricardozanini oops, thanks & fixed!

**Comment by ricardozanini** [comment]
@luck3y I'll run some more integration tests, but it seems that now we have the right libs [according to the docs](https://access.redhat.com/articles/2342881). :heart: 

See:

```
./system/add-ons/keycloak
./system/add-ons/keycloak/org/keycloak
./system/add-ons/keycloak/org/keycloak/keycloak-servlet-oauth-client
./system/add-ons/keycloak/org/keycloak/keycloak-servlet-oauth-client/main/keycloak-servlet-oauth-client-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-adapter-spi
./system/add-ons/keycloak/org/keycloak/keycloak-adapter-spi/main/keycloak-undertow-adapter-spi-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-adapter-spi/main/keycloak-adapter-spi-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-core
./system/add-ons/keycloak/org/keycloak/keycloak-core/main/keycloak-core-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-saml-adapter-subsystem
./system/add-ons/keycloak/org/keycloak/keycloak-saml-wildfly-adapter
./system/add-ons/keycloak/org/keycloak/keycloak-saml-wildfly-adapter/main/keycloak-saml-wildfly-adapter-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-adapter-subsystem
./system/add-ons/keycloak/org/keycloak/keycloak-saml-wildfly-subsystem
./system/add-ons/keycloak/org/keycloak/keycloak-saml-wildfly-subsystem/main/keycloak-saml-wildfly-subsystem-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-authz-client
./system/add-ons/keycloak/org/keycloak/keycloak-authz-client/main/keycloak-authz-client-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-saml-core-public
./system/add-ons/keycloak/org/keycloak/keycloak-saml-core-public/main/keycloak-saml-core-public-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-wildfly-adapter
./system/add-ons/keycloak/org/keycloak/keycloak-wildfly-adapter/main/keycloak-wildfly-adapter-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-adapter-core
./system/add-ons/keycloak/org/keycloak/keycloak-adapter-core/main/keycloak-adapter-core-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-wildfly-elytron-oidc-adapter
./system/add-ons/keycloak/org/keycloak/keycloak-wildfly-elytron-oidc-adapter/main/keycloak-wildfly-elytron-oidc-adapter-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-saml-adapter-core
./system/add-ons/keycloak/org/keycloak/keycloak-saml-adapter-core/main/keycloak-saml-adapter-core-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-common
./system/add-ons/keycloak/org/keycloak/keycloak-common/main/keycloak-common-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-undertow-adapter
./system/add-ons/keycloak/org/keycloak/keycloak-undertow-adapter/main/keycloak-undertow-adapter-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-wildfly-subsystem
./system/add-ons/keycloak/org/keycloak/keycloak-wildfly-subsystem/main/keycloak-wildfly-subsystem-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-saml-undertow-adapter
./system/add-ons/keycloak/org/keycloak/keycloak-saml-undertow-adapter/main/keycloak-saml-undertow-adapter-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-jboss-adapter-core
./system/add-ons/keycloak/org/keycloak/keycloak-jboss-adapter-core/main/keycloak-jboss-adapter-core-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-saml-wildfly-elytron-adapter
./system/add-ons/keycloak/org/keycloak/keycloak-saml-wildfly-elytron-adapter/main/keycloak-saml-wildfly-elytron-adapter-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-saml-core
./system/add-ons/keycloak/org/keycloak/keycloak-saml-core/main/keycloak-saml-core-4.8.7.Final-redhat-00001.jar
./system/add-ons/keycloak/org/keycloak/keycloak-saml-adapter-api-public
./system/add-ons/keycloak/org/keycloak/keycloak-saml-adapter-api-public/main/keycloak-saml-adapter-api-public-4.8.7.Final-redhat-00001.jar
```

**Comment by luck3y** [comment]
@bstansberry Are you OK with how this is set in Jira now? It should be for master.

**Comment by luck3y** [comment]
@bstansberry Any chance of another look at this one? I think we'll want this on the stable branches ASAP.

**Comment by ricardozanini** [comment]
@luck3y @bstansberry sorry to be a pita, but this PR is even more urgent and important to us than the others. It’s a requirement from field/PM to upgrade the client because of CVEs.

**Pull request merged**